app/hooks/published-projects/index.ts

```diff
@@ -18,7 +18,7 @@ export function usePublishedProjects(options: UsePublishedProjectsProps = {}) {
   const {
     search,
     filters = {},
-    sort = '-lastModifiedAt',
+    // sort = '-lastModifiedAt',
```

Comment:
This was commented as database has not this column. A task has been added to JIRA.